FluentWPF/Styles/TextBox.xaml

```diff
@@ -44,7 +89,29 @@
                         <Trigger Property="IsKeyboardFocused" Value="true">
                             <Setter Property="BorderBrush" TargetName="border" Value="{StaticResource TextBox.Focus.Border}"/>
                             <Setter Property="Background" Value="#EEFFFFFF" />
+                            <Trigger.EnterActions>
+                                <BeginStoryboard Storyboard="{StaticResource enterGotFocus}" />
+                            </Trigger.EnterActions>
+                            <Trigger.ExitActions>
+                                <BeginStoryboard Storyboard="{StaticResource exitGotFocus}" />
+                            </Trigger.ExitActions>
                         </Trigger>
+
+
+                        <DataTrigger Binding="{Binding ElementName=Label_Header, Path=Text}" Value="">
+                            <Setter TargetName="Label_Header" Property="Visibility" Value="Collapsed"/>
+                        </DataTrigger>
+
+
+                        <DataTrigger Binding="{Binding Text.Count, RelativeSource={RelativeSource Self}}" Value="0">
```

Comment:
It Works, but I couldn't explain this behavior.
Text doesn't have `Count` property. `Count` is method that is defined in IEnumerable<T>'s extension method.
IDE warns this code, too.
I think PasswordBox's code is more suitable for it.
```xml
<DataTrigger Binding="{Binding RelativeSource={RelativeSource Self}, Path=Text, Mode=OneWay}" Value="">
```